src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -120,7 +131,8 @@ private void EnsureReceiptAvailable()
             {
                 if (!syncConfig.DownloadReceiptsInFastSync || !syncConfig.DownloadBodiesInFastSync)
                 {
-                    throw new InvalidOperationException("Receipt and body must be available for merge to function");
+                    if (_logger.IsError) _logger.Error("Receipt and body must be available for merge to function. The following configs values should be set to true: Sync.DownloadReceiptsInFastSync, Sync.DownloadBodiesInFastSync");
+                    _environment.Exit(1);
```

Comment:
I would create some static ExitCodes class and name those + make unique codes

Reply:
> I would create some static ExitCodes class and name those + make unique codes, so if we get info from users we can actually diagnose it easily

Done :)